charts/internal/cilium/charts/config/templates/configmap.yaml

```diff
@@ -414,14 +414,27 @@ data:
 {{- end }}
 
 {{- if .Values.global.encryption.enabled }}
+{{- if eq .Values.global.encryption.mode "ipsec" }}
```

Comment:
Should we really include ipsec? Did you test it?
Last time I tested, I didn't get it to work in a real cluster. Though, I didn't spend time to debug.
At least, we should document, how to create the required secret.
I would prefer, to omit the ipsec part for now.

Reply:
I would agree to take smaller steps and only add wg for now. Also I could not find a test for ipsec in this PR.